README.md

```diff
@@ -87,7 +87,7 @@ from either OpenAI or Azure OpenAI and to run one of the C#, Python, and Java co
 ### For Python:
 
 1. Go to the Quick start page [here](https://learn.microsoft.com/en-us/semantic-kernel/get-started/quick-start-guide?pivots=programming-language-csharp) and follow the steps to dive in.
-2. You'll need to ensure that you toggle to C# in the the Choose a programming language table at the top of the page.
+2. You'll need to ensure that you toggle to Python in the the Choose a programming language table at the top of the page.
```

Comment:
It seems this whole section is referencing C# language items. @sophialagerkranspandey thoughts?

Reply:
Thanks @moonbox3. I updated the content in this section to align to Python (link to the quick start page and the alt-text on the image).